packages/web-app-files/src/components/Search/List.vue

```diff
@@ -397,31 +397,36 @@ export default defineComponent({
         query['mediatype'] = mediaTypeParams.split('+').map((t) => `"${t}"`)
         updateFilter(mediaTypeFilter)
       }
+      // By definition (KQL spec) OR, AND or (GROUP) is implicit for simple cases where
+      // different or identical keys are part of the query.
+      //
+      // We list these operators for the following reasons nevertheless explicit:
+      // * request readability
+      // * code readability
+      // * complex cases readability
+      let searchTermArray = Object.keys(query).reduce((acc, prop) => {
+        const isArrayValue = Array.isArray(query[prop])
+
+        if (!isArrayValue) {
+          acc.push(`${prop}:${query[prop]}`)
+        }
 
-      return (
-        // By definition (KQL spec) OR, AND or (GROUP) is implicit for simple cases where
-        // different or identical keys are part of the query.
-        //
-        // We list these operators for the following reasons nevertheless explicit:
-        // * request readability
-        // * code readability
-        // * complex cases readability
-        Object.keys(query)
-          .reduce((acc, prop) => {
-            const isArrayValue = Array.isArray(query[prop])
-
-            if (!isArrayValue) {
-              acc.push(`${prop}:${query[prop]}`)
-            }
-
-            if (isArrayValue) {
-              acc.push(`${prop}:(${query[prop].join(' OR ')})`)
-            }
-
-            return acc
-          }, [])
```

Comment:
Please use the `sort` array helper before the `join` to make sure that the scope is at the end (just specify a sort callback function that does nothing else than sorting items starting with `scope:` to the end). Can be called in all cases, even on an empty result after the reducer finished, so you can keep the old structure of the code and only add the sort call.

Reply:
Updated